src/pages/community.tsx

```diff
@@ -1,312 +1,136 @@
-  display: flex;
-  align-items: center;
-  flex-direction: row-reverse;
-  padding-left: 2rem;
-  height: 720px;
-  width: 100%;
-  margin-top: -1px;
-  margin-bottom: 0rem;
-  border-top: 1px solid ${(props) => props.theme.colors.text};
-  border-bottom: 1px solid ${(props) => props.theme.colors.text};
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column-reverse;
-    height: 100%;
-    padding-top: 2rem;
-    padding-left: 0rem;
-    padding-bottom: 2rem;
-  }
-`
+const Divider: React.FC = () => {
+  return <Box my={16} w="10%" h={1} bgColor="homeDivider" />
+}
 
-const SupportContainer = styled.div`
-  background: ${(props) => props.theme.colors.homeBoxPurple};
-  display: flex;
-  align-items: center;
-  flex-direction: row;
-  height: 720px;
-  width: 100%;
-  margin-top: -1px;
-  border-top: 1px solid ${(props) => props.theme.colors.text};
-  border-bottom: 1px solid ${(props) => props.theme.colors.text};
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column-reverse;
-    height: 100%;
-  }
-`
+const Page: React.FC<IProps> = ({ children }) => {
+  return (
+    <Flex direction="column" alignItems="center" w="full" mx="auto">
+      {children}
+    </Flex>
+  )
+}
 
-const LeftColumn = styled.div`
-  margin-right: 2rem;
-  width: 100%;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin: auto 0rem;
-  }
-`
+const ButtonRow: React.FC<IProps> = ({ children }) => {
+  return (
+    <Flex alignItems="flex-start" direction={{ base: "column", md: "row" }}>
+      {children}
+    </Flex>
+  )
+}
 
-const FeatureContent = styled(LeftColumn)`
-  padding: 6rem;
-  height: 100%;
-  width: 100%;
-  margin: 0;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    padding: 2rem;
-  }
-`
+const StyledButtonLink: React.FC<IButtonLinkProps> = ({
+  children,
+  ...props
+}) => {
+  return (
+    <ButtonLink
+      mt={{ base: 4, md: 0 }}
+      ml={{ base: 0, md: 2 }}
+      display="flex"
+      alignItems="center"
+      {...props}
+    >
+      {children}
+    </ButtonLink>
+  )
+}
 
-const LeftColumnContent = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-`
+const RowReverse: React.FC<IProps> = ({ children }) => {
+  return (
+    <Flex
+      direction={{ base: "column-reverse", lg: "row-reverse" }}
+      alignItems={{ base: "center", lg: "normal" }}
+    >
+      {children}
+    </Flex>
+  )
+}
 
-const IntroLeftColumn = styled(LeftColumn)`
-  padding: 6rem;
-  height: 100%;
-  width: 100%;
-  margin: 0;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    padding: 2rem;
-  }
-  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
-    padding: 0rem;
-  }
-`
+const ImageContainer: React.FC<IProps> = ({ children }) => {
+  return (
+    <Flex bg="#f1fffd" h="full" w={{ base: "75%", lg: "full" }}>
```

Comment:
Funny thing, that bg color was failing in the prev implementation and we weren't seeing that color on production. But now that you have done it correctly, we have this result xD
![image](https://user-images.githubusercontent.com/468158/219215710-1cdc158e-b41e-40d6-8384-5db289dd5bb0.png)

So, lets remove that `bg`, was not used anyway.